tests/fixtures/sqlserver/datacontract.yaml

```diff
@@ -0,0 +1,28 @@
+dataContractSpecification: 0.9.2
+id: sqlserver
+info:
+  title: sqlserver
+  version: 0.0.1
+  owner: my-domain-team
+servers:
+  my-dataproduct/sqlserver:
+    type: sqlserver
+    host: localhost
+    port: __PORT__
+    database: test
+    schema: dbo
+    driver: ODBC Driver 18 for SQL Server
+models:
+  my_table:
+    type: table
+    fields:
+      field_one:
+        type: varchar
+        required: true
+        unique: true
+        regex: "[A-Za-z]{2}-\\d{3}-[A-Za-z]{2}$"
```

Comment:
should probably be pattern

Reply:
Hey Simon, how do you mean?

Reply:
regex is not a known field in the data contract spec, should be named pattern, I think.

Reply:
a then it is wrong there as well. Good catch! :-)